modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -58,7 +58,7 @@ public bool LogVisible
         }
 
         // TODO Use List once we have proper serialization.
-        private readonly Array<BuildIssue> _issues = new Array<BuildIssue>();
+        private Godot.Collections.Array _issues = new();
```

Comment:
I removed `readonly` because of the new way we do serialization. We generate a deserialize method now, instead of using reflection. With reflection, it was possible to bypass `readonly`. The only way we could do this from code is if the deserialize method was a constructor overload, which can assign the initial value, but I left that for future consideration.

Reply:
I re-added the generic version.